src/Utils.vala

```diff
@@ -168,4 +168,14 @@ namespace Power.Utils {
         ngettext ("%lld minute until empty", "%lld minutes until empty", 0);
         ngettext ("%lld second until empty", "%lld seconds until empty", 0);
     }
+
+    public int handle_scroll (Gdk.EventScroll e) {
+        int step = 10;
+        if (e.direction == Gdk.ScrollDirection.UP) {
+           return step;
+        } else if (e.direction == Gdk.ScrollDirection.DOWN) {
+            step = -step;
+        }
+        return step;
+    }
```

Comment:
This can be simplified to:

```
    public int handle_scroll (Gdk.EventScroll e) {
        int step = 10;
        if (e.direction == Gdk.ScrollDirection.DOWN) {
            step = -10;
        }
        return step;
    }
```

or even

```
 public int handle_scroll (Gdk.EventScroll e) {
    return (e.direction == Gdk.ScrollDirection.UP ? 10 : -10);
 }
```

Also the figure 10 should be a constant  - Power.Utils.STEP.